src/shared/types/wizard/index.ts

```diff
@@ -94,6 +94,7 @@ export interface CommonSharedExtraSettings {
     navigatorSettings?: NavigatorSettings;
     enableGPTInsights?: boolean;
     labelsPosition?: LabelsPositions;
+    qlAutoExecuteChart?: 'on' | 'off';
```

Comment:
Perhaps it is worth dividing the parameters into Wizard and QL ? To have something like `WizardChartSetting | QlChartSetting`.

In this case, you will not need to use the ql prefix.

Reply:
Let's do the refactoring in the next PR